src/test/java/it/gov/pagopa/reminder/MessageKafkaConsumerTest.java

```diff
@@ -87,13 +88,14 @@ public void MockSchedulerNotifyIntegrationReminderKafkaConsumerTest() throws Jso
     
     public void MockSchedulerNotifyIntegrationPaymentUpdatesKafkaConsumerTest(String contentType, String contentType2, String source) {
     	consumer = (PaymentUpdatesKafkaConsumer) ApplicationContextProvider.getBean("paymentUpdatesEventKafkaConsumer");
```

Comment:
Why do we need ApplicationCtx to access this kind of bean in here?

Reply:
Fixed.